js/SplitButton/Easy-customization.md

```diff
@@ -857,7 +857,7 @@ Configure the **CSS** styles to apply on buttons.
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img5.png) 
+![Theme Support in JavaScript Split Button](/js/SplitButton/Easy-customization_images/Easy-customization_img5.png) 
```

Comment:
After the first word, the second word should be in lower case. Check in all places

Reply:
Updated